.github/workflows/github-release.yml

```diff
@@ -0,0 +1,57 @@
+name: github-release
+on:
+  push:
+    tags:
+    - "v[0-9]+.[0-9]+.[0-9]+"
+jobs:
+  github-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        # We neeed to fetch the entire history as conventional-changelog needs
+        # access to any number of git commits to build the changelog.
+        with:
+          fetch-depth: 0
```

Comment:
I guess the alternative is to use a specific number and hope that we never actually cross that threshold between releases haha. Not sure that's worth doing, though.

Reply:
Yeah I think we just want to download it all, If grabbing the entire repo takes longer for releases to be generated, it isn't the end of the world. As long as it works.

Reply:
Yeah, if we see it ends up being slow, we can cap this at that point.